.github/workflows/test.yml

```diff
@@ -14,32 +14,29 @@ env:
   VITE_CLERK_USER: ${{ vars.VITE_CLERK_USER }}
 
 jobs:
-  test-node-16:
+  test-old:
```

Comment:
What's the meaning of this change? 👀

Reply:
I gave up trying to make the tests work with Node 16, because it requires a `require('crypto')` in the library, and that might break bundlers of various sorts. In practice, Node 16 should work when connecting to Neon, but might break if running your own WebSocket proxy.